misk-redis/src/main/kotlin/misk/redis/RedisModule.kt

```diff
@@ -6,9 +6,8 @@ import com.google.inject.Singleton
 import misk.inject.KAbstractModule
 import redis.clients.jedis.Jedis
 
-class RedisModule(val config: RedisConfig): KAbstractModule() {
+class RedisModule : KAbstractModule() {
   override fun configure() {
-    bind<RedisConfig>().toInstance(config)
```

Comment:
did result in a double binding error? can we add a test that catches this bug?

Reply:
For sure, I'll make sure to add a test that uses this module to catch this